cinema/serializers.py

```diff
@@ -125,10 +126,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
+            ValidationError,
         )
         return data
```

Comment:
The `validate` method in `TicketSerializer` is a little bit confusing. It's not clear what the `validate_ticket` method does just by looking at this code. It would be helpful to add a comment explaining what this validation does. Furthermore, it's generally better to put complex validation logic into the model's `clean` method or a custom validator, instead of the serializer.